masl.src.html

```diff
@@ -99,44 +99,44 @@ <h3>Single or Multiple Resources</h3>
             map are complete paths that MUST start with <code>/</code> and the values are metadata
             objects that MUST have an <code>src</code> field pointing to the resource's CID and SHOULD have a
             <code>mediaType</code> field giving its MIME type, along with other HTTP headers.
+            Any other properties present which do not map to HTTP headers MUST be ignored.
```

Comment:
```suggestion
```

I don't think that's true. We might create new metadata properties in other specs.

Reply:
MUST NOT be sent as HTTP headers? if someone throws `"foo":"bar"` in there, I wouldn't want an HTTP server to send header "foo" (or X-foo?) just because it assumes all props are headers...

Reply:
oh nevermind leaving "SHOULD NOT reflect" later on covers this